package.json

```diff
@@ -130,10 +130,10 @@
     ]
   },
   "dependencies": {
-    "@csstools/css-parser-algorithms": "^2.2.0",
+    "@csstools/css-parser-algorithms": "^2.3.0",
     "@csstools/css-tokenizer": "^2.1.1",
-    "@csstools/media-query-list-parser": "^2.1.1",
-    "@csstools/selector-specificity": "^2.2.0",
+    "@csstools/media-query-list-parser": "^2.1.2",
+    "@csstools/selector-specificity": "^3.0.0",
```

Comment:
License was changed from `CC0-1.0` to `MIT-0` for `@csstools/selector-specificity`.

Other packages from `@csstools` already had an `MIT` license.